### ISSUE TITLE
Make it easy for the user to gather all relevant logs

### ISSUE THREAD

**Issue by thallgren** [telepresenceio/telepresence#1996]
Provide a subcommand that helps the user to collect and zip the log output from the root and user daemons, the traffic manager, and if relevant from intercepting traffic agents. Example:

Collect all logs (best effort is made to guess the traffic-agent)
```console
➜  telepresence gather-logs --output /tmp/tplogs.zip
```

Collect logs for a specific daemon
```console
➜  telepresence gather-logs --root-daemon --output /tmp/tplogs.zip
```

Collect logs for the traffic manager
```console
➜  telepresence gather-logs --traffic-manager --output /tmp/tplogs.zip
```

Collect logs from the traffic agent(s) for a workload.
```console
➜  telepresence gather-logs --traffic-agent=<workload name> --output /tmp/tplogs.zip
```

We must also provide a `--anonymise` option that automatically redacts sensitive information from the logs.

**Comment by musaib072** [comment]
can you tell me in which folder or file this issue is present . I want to work on it

**Comment by efunk** [comment]
Resolved in Telepresence 2.4.4 GA
